src/components/instrument/CreateUpdateInstrument.tsx

```diff
@@ -31,13 +33,20 @@ const CreateUpdateInstrument: React.FC<CreateUpdateInstrumentProps> = ({
 }) => {
   const classes = useStyles();
   const { api, isExecutingCall } = useDataApiWithFeedback();
+  const { usersData } = useUsersData({
+    userRole: UserRole.INSTRUMENT_SCIENTIST,
+  });
 
+  if (!usersData) {
+    return <UOLoader />;
+  }
   const initialValues = instrument
     ? instrument
     : {
         name: '',
         shortCode: '',
         description: '',
+        managerUserId: -1,
```

Comment:
Not sure why is this -1 as initial value

Reply:
When creating new instrument this must be set initialy to an invalid userId, because the creator has go specify it